### ISSUE TITLE
Vagrant box bento/ubuntu-16.04 is unusable with Linux Mint Sarah hosting

### ISSUE THREAD

**Issue by znmeb** [hackoregon/linux-laptop-setup#32]
This is a long story. I started down the path of trying to install my Linux setup scripts in a Vagrant box. The base box seemed OK, but everything I tried beyond that locked my host system up.

The host system in question is an 8-core 32 GB workstation running Linux Mint 18. I went through the usual troubleshooting process. Tried VirtualBox 5.0 instead of 5.1, older versions of Vagrant, etc. Still getting lockups.

I gave up on bento/ubuntu-16.04 and tried ubuntu/xenial64. The lockups are gone! So I went hunting for the source of the bento boxes. As near as I can tell, this is where they live on GitHub: <https://github.com/chef/bento>. 

So I looked at the open issues and found a few other Hack Oregonians have encountered. <https://github.com/chef/bento/issues> Reading those issues and the interactions with various versions of VirtualBox and Vagrant has ***reduced*** my confidence in the viability of these boxes as a base.

I'm going to proceed with ubuntu/xenial64 for at least this release. When I get around to testing others' Vagrant boxes, if I get lockups, I'm going to recommend we abandon bento as a base box.

Watch this space.

**Comment by znmeb** [comment]
Well ... for openers, the Ubuntu base box only has a 10 GB hard drive :-1:

**Comment by scottslewis** [comment]
It would be nice if we could find a box that allows easy installation on all linux, all windows, all macs.  Note that the mini-conda install https://github.com/hackoregon/hack-university-data-architecture/blob/master/toolchain/vagrant/Vagrantfile currently uses bento/ubuntu too, so if your work on this can be applied there it would be especially helpful.   Last thing we want is to have a whole  bunch of box-specific problems experienced by folks next week.

**Comment by znmeb** [comment]
Yeah ... I'm in the process of building a base box now according to the instructions at https://www.vagrantup.com/docs/boxes/base.html and https://www.vagrantup.com/docs/virtualbox/boxes.html. I'm doing the first one manually; it can be automated but with luck we'll only have to do it once before the install-a-thon. When we start deploying to AWS we'll have much better tools than Vagrant / VirtualBox.
